public/src/components/campaign/campaign_list.jsx

```diff
@@ -15,21 +15,15 @@ export default class CampaignList extends Component {
   render() {
     const {
       account_info,
-      addCampaignToUser,
       all_campaigns,
       auth,
       history,
-      setCurrentCampaign
+      updateCampaignStatus
     } = this.props;
     const thisPage = 'Campaign';
     const { admin_campaigns, volunteer_campaigns } = buttons_obj;
     const { redirect_path } = campaignHeaders;
 
-    function handleClick(id, historyObj, campaign) {
```

Comment:
I assume these were unused in the list view for some reason?

Reply:
@nguyenaiden isn't this necessary for assigning users to campaigns in the volunteer campaign view?

Reply:
That function was mapped to the button group, but it was never used since the handler function for ```admin_campaigns``` in `button_objs.jsx`  didn't map to any handleClick until this PR.